src/app/modules/shared/handler/data-object-builder.handler.ts

```diff
@@ -44,16 +44,18 @@ export class DataObjectBuilderHandler {
       DataObjectBuilderHandler.addContext(component, dataObject, templateJsonObj, buildingMode);
       // A UUID need to be assigned inorder to model to validate. This UUIDs should be overwritten by backend later.
```

Comment:
WIll be good to move this comment down where randomAtID function is called? So it's not confusing here

Reply:
Yes, I just removed it